packages/@aws-cdk/alexa-ask/package.json

```
@@ -50,11 +50,15 @@
     "cfn2ts": "cfn2ts",
     "build+test+package": "npm run build+test && npm run package",
     "build+test": "npm run build && npm test",
-    "compat": "cdk-compat"
+    "compat": "cdk-compat",
+    "gen": "cfn2ts"
```

Why don't we put the arguments to cfn2ts here as well? Let's make this
```suggestion
    "gen": "cfn2ts --scope=Alexa::Ask"
```
etc.

---

We don't need it, `cfn2ts` already gets this from the `pacakge.json` file.

---

I know, but it gets it from the `cdk-build` section. Don't you agree that's kinda confusing because we're not using `cdk-build` to invoke `cfn2ts` anymore?

In any case, fine.

---

I agree it's confusing, as evident by having to read through `cfn2ts` code to understand why calling it without the scope name even works. I think we can add it to the `gen` directive for clarity, not sure about removing the logic from `cfn2ts`, it is a private package so technically we wont be breaking anyone.